.github/workflows/publish-release.yml

```diff
@@ -95,7 +95,7 @@ jobs:
           ORG_GRADLE_PROJECT_signingInMemoryKeyPassword: ${{ secrets.SIGNING_PASSWORD }}
 
       - name: Upload Maestro XCUiTest Driver
-        run: ./gradlew clean :maestro-xcuitest-driver:publish --no-daemon --no-parallel
+        run: ./gradlew clean :maestro-ios-driver:publish --no-daemon --no-parallel
```

Comment:
Wouldn't this be confused with the other directory where we have server code?

There would be maestro-ios-driver and maestro-driver-ios (swift one) in codebase, which is confusing for me. Sounds like same.

Reply:
From the description:

> Rename:
>
> Module maestro-xcuitest-driver to maestro-ios-driver (contains the code to manage iOS Simulator and talk to the XCTest runner)
> Module maestro-driver-ios to maestro-ios-xctest-runner (contains the xctest runner code)

Reply:
Guys, this is both in the description and in the changes made. If there are any doubts, use `git switch rename_simctl` and see for yourself.